bundles/action/org.openhab.action.telegram/README.md

```diff
@@ -112,3 +112,17 @@ end
 Do not use username/password in url like in this example `http://<username>:<password>@server/image.png`; pass the credentials to the `sendTelegramPhoto` method instead.
 
 `http` and `https` are the only protocols allowed.
+
+In case you want to send a base64 jpeg or png image.

```

Comment:
I know this suggestion does not conform to the way the rest of the article is written, but please consider, "If you would like to send a base64..."